src/statsig/constants.ts

```diff
@@ -89,4 +89,11 @@ export const DynamicConfigs = {
       blockedNouns: [] as string[],
     },
   },
+  [StatsigDynamicConfigs.WALLET_NETWORK_TIMEOUT]: {
+    configName: StatsigDynamicConfigs.WALLET_NETWORK_TIMEOUT,
+    defaultValues: {
+      default: 15,
+      cico: 30,
```

Comment:
these are really long. I doubt most users would wait 30 seconds for something before leaving the app

Reply:
how about 10 for both?

Reply:
or 10 for default and 15 for cico?

Reply:
I've seen some getQuotes requests take 10+ seconds on my simulator. I suspect this would be higher for real users especially in different regions especially when there's a cold start. Although I think 15 should work without cold starts for most regions.
With regards to default, I just went with what the current default was on the fetchWithTimeout method. Although right now this default isn't used anywhere yet, so we could set it to 10.